### PR TITLE
Update to QPrompt v1.1.0

### PR DESCRIPTION
**Pull request by Cuperino**

### CODE REVIEW

**Comment by Cuperino** [comment]
Pushed missing git tag.
bot, build

**Comment by Cuperino** [comment]
Build [103673](https://buildbot.flathub.org/#/builders/32/builds/103673) was triggered after [103668](https://buildbot.flathub.org/#/builders/32/builds/103668). Testing the program on arm64 and x86_64 yielded perfect results on both architectures.

Proceeding to merge pull request.

File: com.cuperino.qprompt.json [hunk 1]
```diff
@@ -23,14 +23,26 @@
    ],
    "modules": [
       {
+         "name": "qhotkey",
+         "buildsystem": "cmake-ninja",
+         "sources": [
+            {
+               "type": "git",
+               "url": "https://github.com/Skycoder42/QHotkey.git",
+               "branch": "master",
```
Comment (hfiguiere) [inline com.cuperino.qprompt.json:32]: you don't want a branch here it will break the build at the next commit.

Comment (Cuperino) [reply com.cuperino.qprompt.json:32]: I see. Thanks for the tip and fix.

**Pull request merged**